onnx/test/helper_test.py

```diff
@@ -883,6 +889,20 @@ def test_tensor_dtype_to_field_bfloat16(self) -> None:
         )
 
 
+@pytest.mark.parametrize(
+    "attr_type",
+    AttributeProto.AttributeType.values(),
+)
+def test_attr_type_to_str(attr_type):
+    result = helper._attr_type_to_str(attr_type)
+    assert result == AttributeProto.AttributeType.keys()[attr_type]
```

Comment:
This test is repeating the logic: it seems more desirable to spell the expected string out in the parameterization.

Reply:
I think I've addressed this in the test